packages/@o3r/dynamic-content/src/stores/index.ts

```diff
@@ -3,5 +3,6 @@ export * from './asset-path-override/asset-path-override.module';
 export * from './asset-path-override/asset-path-override.reducer';
```

Comment:
Should not export `./asset-path-overrive/index` instead? (And export the files from this barrel)?